tools/doc/html.js

```diff
@@ -390,3 +418,14 @@ function getId(text) {
   }
   return text;
 }
+
+const numberRe = /^(\d*)/;
+function versionSort(a, b) {
+  a = a.trim();
+  b = b.trim();
+  let i = 0;  // common prefix length
+  while (i < a.length && i < b.length && a[i] === b[i]) i++;
+  a = a.substr(i);
+  b = b.substr(i);
+  return +b.match(numberRe)[1] - +a.match(numberRe)[1];
```

Comment:
@bnoordhuis I actually couldn’t get this code to fail, it would just treat `v0.12.x` as 0 (assuming the common prefix is `v0.12.`). That seems somewhat reasonable to me.